programs/disks/Path.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+#include "Path.h"
+
+namespace DB
+{
+
+Path::Path(std::string def_disk_name, std::string to_parse)
+{
+    path = "";
+    disk_name = def_disk_name;
+
+    if (to_parse.find(':') == std::string::npos)
```

Comment:
if(auto pr = to_parse.find(':'); pr == std::string::npos)
You can avoid second find call